src/components/HomeBlocks/LeaderboardsHomeBlock.tsx

```diff
@@ -30,30 +32,44 @@ export const LeaderboardsHomeBlock = ({ homeBlock }: Props) => {
   const { leaderboards } = homeBlock;
   const metadata = homeBlock.metadata as HomeBlockMetaSchema;
 
-  console.log(homeBlock.leaderboards);
-
   return (
     <HomeBlockWrapper className={classes.root}>
       {metadata?.title && (
-        <Group position="apart" align="center" pb="md">
-          <Title>{metadata?.title}</Title>
+        <Group position="apart" align="center" pb="md" style={{ flexWrap: 'nowrap' }}>
```

Comment:
There's this handy prop for nowrap on Group component 👀 
```suggestion
        <Group position="apart" align="center" pb="md" noWrap>
```